riptide-stream/src/test/java/org/zalando/riptide/stream/EnforceCoverageTest.java

```diff
@@ -37,14 +37,14 @@
 
     @Test
     public void shouldUseStreamsConstructor() {
-        new Streams();
+        assertNotNull(new Streams());
     }
 
     @Test
     public void shouldUseStreamConverterConstructor() {
-        Streams.streamConverter();
-        Streams.streamConverter(null);
-        Streams.streamConverter(null, null);
+        streamConverter();
```

Comment:
On a black box, yes! But this is a white box and I know testing this would be highly redundant or just checking for assertNotNull as above.